views/update/update-layout.html

```diff
@@ -20,6 +20,7 @@
 {% from "govuk/components/select/macro.njk"              import govukSelect %}
 {% from "govuk/components/warning-text/macro.njk"        import govukWarningText %}
 {% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+{% from "web-security-node/components/csrf-token-input/macro.njk" import csrfTokenInput %}
```

Comment:
Looks like this PR is for the registration journey but this change relates to Update. If possible, I'd revert this and make the change when we do the PR for the Update journey.

Reply:
@mwestacott Without this, 2 GET tests `"render the beneficial owner statements page with Update data"`on line 106 and 156 fail on beneficial.owner.statements.spec.ts

Reply:
Yep, sounds fine in that case.